### PR TITLE
dpkg-deb: add page

### PR DESCRIPTION
**Pull request by eimanip**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

P.S. I would really appreciate it if you add `hacktoberfest-accepted` label to this pull request after it's approved according to [Hacktoberfest guidelines](https://hacktoberfest.digitalocean.com/hacktoberfest-update)

### CODE REVIEW

File: pages/linux/dpkg-deb.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# dpkg-deb
+
+> Pack, unpack and provide information about Debian archives.
+
+- Information about a package:
+
+`dpkg-deb --info {{path/to/file.deb}}`
+
+- Show the package's name and version on one line:
```
Comment (navarroaxel) [inline pages/linux/dpkg-deb.md:9]:
```suggestion
- Display the package's name and version on one line:
```

File: pages/linux/dpkg-deb.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# dpkg-deb
+
+> Pack, unpack and provide information about Debian archives.
+
+- Information about a package:
```
Comment (navarroaxel) [inline pages/linux/dpkg-deb.md:5]:
```suggestion
- Display information about a package:
```

File: pages/linux/dpkg-deb.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Show the package's name and version on one line:
+
+`dpkg-deb --show {{path/to/file.deb}}`
+
+- List the package contents:
+
+`dpkg-deb --contents {{path/to/file.deb}}`
+
+- Extract package contents into a directory:
```
Comment (navarroaxel) [inline pages/linux/dpkg-deb.md:17]:
```suggestion
- Extract package's contents into a directory:
```

File: pages/linux/dpkg-deb.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Information about a package:
+
+`dpkg-deb --info {{path/to/file.deb}}`
+
+- Show the package's name and version on one line:
+
+`dpkg-deb --show {{path/to/file.deb}}`
+
+- List the package contents:
```
Comment (navarroaxel) [inline pages/linux/dpkg-deb.md:13]:
```suggestion
- List the package's contents:
```

File: pages/linux/dpkg-deb.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# dpkg-deb
+
+> Pack, unpack and provide information about Debian archives.
```
Comment (navarroaxel) [inline pages/linux/dpkg-deb.md:3]:
```suggestion
> Pack, unpack and provide information about Debian archives.
> More information: <https://manpages.debian.org/buster/dpkg/dpkg-deb.1.en.html>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @eimanip, welcome and thanks for your submission. I left some comments.

**Review by navarroaxel — APPROVED** [review]

**Comment by eimanip** [comment]
Hey @navarroaxel ,
Thanks for your great suggestions. I applied all of them.
By the way, I submitted this pull request by inspiring from the `dpkg` page. Some of your comments (including having a "more information" link) could be also applied for `dpkg`. I would be happy to also apply similar changes for `dpkg` page and include them in this pull request or a new one.

**Comment by navarroaxel** [comment]
@eimanip Yeah, please send a new PR for the edition.

File: pages/linux/dpkg-deb.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- List the package's contents:
+
+`dpkg-deb --contents {{path/to/file.deb}}`
+
+- Extract package's contents into a directory:
+
+`dpkg-deb --extract {{path/to/file.deb}} {{path/to/directory}}`
+
+- Create a package from the package directory:
```
Comment (sbrl) [inline pages/linux/dpkg-deb.md:22]: Perhaps this will read better?
```suggestion
- Create a package from a specified directory:
```

**Review by sbrl — APPROVED** [review]
Thanks, @eimanip!

**Comment by eimanip** [comment]
Thank you for approving the pull request.
Would you mind adding `hacktoberfest-accepted` label to this pull request if it's ok to you? ([More info](https://hacktoberfest.digitalocean.com/))

**Pull request merged**